### ISSUE TITLE
Port to python3

### ISSUE THREAD

**Issue by marado** [marado/stars-to-addresses#4]

**Comment by Yoshi325** [comment]
@marado I would be interested in helping with this for Hacktoberfest. Would you assign the issue to me?

**Comment by marado** [comment]
There you go!

**Comment by Yoshi325** [comment]
@marado do you have a specific version of Python 3 you are wanting to target?

**Comment by marado** [comment]
Not really: just something still maintained... If it helps, I'll probably test your PR with Python 3.8.2.

**Comment by Yoshi325** [comment]
@marado I put up two options for you. Please feel free to pick between them as it suits you.
